lib/src/extensions/image.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:typed_data';
+import 'dart:ui';
+
+import '../flame.dart';
+import 'color.dart';
+
+export 'dart:ui' show Image;
+
```

Comment:
I would add some comments here regarding:

this is *very* slow. do **not** do this every tick in your game, or your device, emulator, computer, city's power grid and the www will explode.